src/data/reducers/assets.js

```diff
@@ -5,19 +5,44 @@ import { assetLoading } from '../constants/loadingTypes';
 
 import { addLoadingField, removeLoadingField, toggleLockAsset } from './utils';
 
-const initialState = {
+const filtersInitial = {
+  assetTypes: {},
+};
+
+const metaDataInitial = {
+  start: 0,
+  end: 0,
   page: 0,
   pageSize: 50,
-  assetTypes: {},
+  totalCount: 0,
+};
+
+const sortOrderInitial = {
   sort: 'date_added',
   direction: 'desc',
 };
 
+const requestInitial = {
+  ...filtersInitial,
+  ...metaDataInitial,
+  ...sortOrderInitial,
+};
+
+
+const filters = (state = filtersInitial, action) => {
```

Comment:
Filters is a slice of the Redux state that represents filters on the assets. It represents the actual, current state of the table's filters.